### PR TITLE
Bumped Elixir to 1.5.3.

### PR DESCRIPTION
**Pull request by dinsley**

### CODE REVIEW

**Comment by dinsley** [comment]
[Elixir Upgrade to 1.5.3](https://trello.com/c/qvQ36LN5/250-elixir-upgrade-to-153)

**Comment by dinsley** [comment]
[Elixir Upgrade to 1.5.3](https://trello.com/c/qvQ36LN5/250-elixir-upgrade-to-153)

**Review by snarfmason — APPROVED** [review]

**Comment by snarfmason** [comment]
I don't have write access to this repo, so I can't merge this.

**Pull request merged**
